app/endpoints/campaign.py

```diff
@@ -37,13 +44,21 @@
 )
 async def get_sections(
     db: AsyncSession = Depends(get_db),
-    user: models_core.CoreUser = Depends(is_user_a_member_of(GroupType.AE)),
+    user: models_core.CoreUser = Depends(is_user_a_member),
 ):
     """
     Return sections in the database as a list of `schemas_campaign.SectionBase`
 
-    **The user must be a member of the group AE to use this endpoint**
+    **The user must be a voter to use this endpoint**
     """
+    voters = await cruds_campaign.get_voters_list(db)
+    if voters:
```

Comment:
I would be in favor of blocking every user if no allowed groups where set. This will prevent someone from opening a vote while forgetting to configure Voters groups